aligned_accessor/aligned_accessor.md

```diff
@@ -0,0 +1,300 @@
+
+---
+title: "A free function linear algebra interface based on the BLAS"
+document: P1673
```

Comment:
Need to update this

Reply:
Fixed; thanks! I'll get the paper number later today.